target/linux/ar71xx/base-files/lib/ar71xx.sh

```diff
@@ -963,6 +963,9 @@ ar71xx_board_detect() {
 	*TEW-823DRU)
 		name="tew-823dru"
 		;;
+	*"TL-WR942N v1")
```

Comment:
Please, follow the general rule and keep all boards in alphabetical order in all scripts.